Makefile

```diff
@@ -307,6 +309,11 @@ prd/geoadmin.appcache: src/geoadmin.mako.appcache \
 	    --var "api_url=$(API_URL)" \
 	    --var "public_url=$(PUBLIC_URL)" $< > $@
 
+prd/cache/:
+	mkdir -p $@
+	curl -q -o prd/cache/services http:$(API_URL)/rest/services
+	$(foreach lang, $(LANGS), curl -q -o prd/cache/layersConfig.$(lang) http:$(API_URL)/rest/services/all/MapServer/layersConfig?lang=$(lang);)
+
```

Comment:
I think we need a clean cache command as well, to remove old files. Or by default, remove the created files when make clean is called.